src/CanonicalizeGPUVars.cpp

```diff
@@ -46,8 +50,8 @@ class CountGPUBlocksThreads : public IRVisitor {
         nl += dl;
         nt += dt;
 
-        // Update the high water mark.
-        nblocks = std::max(nb, nblocks);
+        // Update the maximum counter values seen.
+        Nblocks = std::max(nb, nblocks);
```

Comment:
this seems like a typo? `Nblocks` -> `nblocks`

Reply:
There's some key command in emacs that capitalizes a word that I accidentally hit from time to time.